src/style.css

```
@@ -0,0 +1,28 @@
+body {
+  margin: 0;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  margin-left: 4px;
+  background-position: 50%;
+  background-image: url("./images/star.svg");
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+.stars--1 .stars__star:first-child,
```

```suggestion
.stars--1 .stars__star:nth-child(-n + 1),
```

Replace with this code to make all code the same style.